fabric-keybindings-v0/src/main/java/net/fabricmc/fabric/api/client/keybinding/FabricKeyBinding.java

```diff
@@ -26,10 +30,40 @@
  * <p>
  * *ALL* instantiated FabricKeyBindings should be registered in
  * {@link KeyBindingRegistry#register(FabricKeyBinding)}!
+ * </p>
+ * <pre><code>
+ * FabricKeyBinding.Builder builder = new FabricKeyBinding.builder();
+ * FabricKeyBinding left = builder
+ *			.id(new identifier("example", "left"))
```

Comment:
A little nitpicky, but this should be capitalized :)
```suggestion
 *			.id(new Identifier("example", "left"))
```